### PR TITLE
chore(lint): change vis typing to external module

### PR DESCRIPTION
**Pull request by alexeagle**
followup from https://github.com/DefinitelyTyped/DefinitelyTyped/pull/13092

### CODE REVIEW

File: vis/tslint.json [hunk 1]
```diff
@@ -1,7 +0,0 @@
-{
```
Comment (andy-ms) [inline vis/tslint.json]: If you delete this file we won't run tests for linting any more. Please keep this, but limit it to just `{ "extends": "../tslint.json" }`.

**Comment by seveves** [comment]
As far as I can see there is nothing I would change. At the moment I am still using my own fork of the typings-vis repository but I will see if I can change easily to this one. Thank you!

File: vis/tslint.json (hunk 1 above)
Comment (alexeagle) [inline vis/tslint.json]: done

**Comment by mmaitre314** [comment]
Apologies for missing the updates on the other PR, Github's email notifications got drowned in noise.

A couple of questions about this PR:
- Are there docs/specs on ` export as namespace`? I missed that in TS' main doc.
- Does `export as namespace vis` take all the declarations in the file and moves them under the `vis` namespace (i.e. removing the `vis` namespace is not going to pollute the global namespace with a bunch of types)?

Assuming that's the case, PR LGTM.

**Comment by andy-ms** [comment]
There's a section called "UMD Modules" [here](http://www.typescriptlang.org/docs/handbook/modules.html).

**Comment by andy-ms** [comment]
Want to do a `types-2.0` -> master merge today, and that will include vis, so I'll merge this and then do the rest of the linting myself.

**Pull request merged**